userdocs/src/usage/nodegroup-customize-dns.md

```diff
@@ -0,0 +1,38 @@
+# Nodegroups with custom DNS
+
+There are two ways of overwriting the DNS server IP address used for all the internal and external DNS lookups. This
+is the equivalent of the `--cluster-dns` flag for the `kubelet`.
+
+The first, is through the `clusterDNS` field. [Config files](../schema) accept a `string` field called
```

Comment:
```suggestion
The first, is through the `clusterDNS` field. [Config files](../schema) accepts a `string` field called
```

Reply:
Ups, left this one out but added it. :) Thank you!